spec/models/hospital_spec.rb

```diff
@@ -4,7 +4,7 @@
   end
 
   describe 'Validations' do
-    it { is_expected.to validate_presence_of :name }
+    it { should validate_uniqueness_of(:name) }
```

Comment:
Please do NOT mix in old should` syntax in rspec. Do more research. You can use the `validate_uniqueness_of` matcher with the `is_expected.to` syntax.